subprojects/docs/src/samples/userguide/files/copy/kotlin/build.gradle.kts

```diff
@@ -253,7 +253,7 @@ task<Zip>("distApp") {
 
 // tag::shared-copy-patterns[]
 val webAssetPatterns = Action<CopySpec> {
-    include("**/*.html', '**/*.png', '**/*.jpg")
+    include("**/*.html", "**/*.png", "**/*.jpg")
```

Comment:
Duh! How couldn't I see that?

Reply:
I didn't see it either at first sight, yay for test coverage!